apps/ngrx-1/src/app/store/status/status.effects.ts

```diff
@@ -5,13 +5,14 @@ import { combineLatest, concatMap, map } from 'rxjs';
 import { selectActivities } from '../activity/activity.selectors';
 import { selectUser } from '../user/user.selectors';
 import * as StatusActions from './status.actions';
+import * as AppActions from '../app.actions';
 import { Status } from './status.model';
 
 @Injectable()
 export class StatusEffects {
   loadStatuses$ = createEffect(() => {
     return this.actions$.pipe(
-      ofType(StatusActions.loadStatuses),
+      ofType(StatusActions.loadStatuses, AppActions.initStore),
```

Comment:
This effect should not exist. You are only transforming data already present inside the store => this should tell you that this should be inside a selector function. 

Effects are only great for doing asynchronous task, not transforming data. 

this is the primary goal of this challenge